message_ix_models/model/material/__init__.py

```diff
@@ -805,3 +804,50 @@ def make_xls_input_vc_able(context, files):
         #     print(path, fname)
         # message_ix_models.model.material.util.excel_to_csv(files)
     return
+
+
+@cli.command("test_calib", hidden=True)
+@click.option("--scenario_name", default="NoPolicy")
```

Comment:
The top-level `mix-models` CLI has `--model` and `--scenario` options. Values for these are automatically handled and stored on the Context.

Rather than duplicate those, check the `context` and—if they are not set—fill in a default within the body of the command function.